### DISCUSSION TITLE
Thinking Cap #2 💡

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/24SU-ASIAAM-191A#32] [Thinking Caps]
## Due 7/8 before class at 3:15pm PST

> ## How can participatory mapping and critical cartography be improved?

Post your thinking cap response and reply to others in this thread. 

##### Refer to this [documentation](https://albertkun.github.io/24SU-ASIAAM-191A/help/thinking_caps/) if you need a reminder on how to complete the assignment.

**Comment by giaari15** [comment]
Participatory and critical cartography can be improved through shifting the use and handling in making maps and such away from foreign researchers and governments and directly by the people. As exemplified by the Participatory Mapping reading, it is the arrogance and ignorance of researchers that go to local villages and view these villages and its people as mere subjects which led to the slow build to even using participatory mapping. It is also this arrogance and ignorance that lead to the list of ethical problems brought up in the article, like taking away time, extracting information, and more. To address these ethical issues, mapping should be done by the local people and stay localized unless it is the local people that wish to spread that information, and it will be best used by the local people. It is western hubris to believe that the local people need the aid of western professionals as the purposes of mapping, like social, health, and mobility mapping, are made to serve the local people.

**Reply by stinest** [reply]
I wholeheartedly agree with each point you brought up. It is almost every misrepresentation of local communities that comes from outsiders. Their foreign arrogance _and_ ignorance in mapping makes it that much more important that efforts must remain localized to preserve its accuracy and empower its people.

**Reply by jasminereyz** [reply]
I agree with your suggestions that maps should stay localized, since no one will know an area better than locals. I think it can be beautiful when locals come together and point out smaller things that those in power would never be inclined to mention in a map. With this, the likelihood of misinformation being spread is lessened!

**Reply by pwinnie2026** [reply]
Yes! I agree with the points that you brought up, especially how you talked about western hubris. Especially living in the US right now, this idea that western countries or "first-world countries" are always the ones that are more capable of assisting other countries in aid is a sentiment I have heard throughout. However, a lot of the time, the ethics concerns that the readings bring up are not addressed or considered. When there are mess-ups, oftentimes there is feigned ignorance and lack of accountability when they should be adjusting their efforts in collaboration with the community and the experiences of the locals.

**Comment by stinest** [comment]
Both participatory mapping and critical cartography can be improved by addressing researchers' behavior and ethics. Beyond overall skepticism and neglect in the proper use of participatory methodologies, Chamber's paper elaborates on the detrimental impact of unethical practices, such as extracting information to use against communities and/or for the sole benefit of outsiders, almost always for financial gain. I believe that improvements can be made by creating a standard and pushing forward a deeper understanding of morals, ensuring that these mapping practices are guided by respect and the genuine interests of the communities involved.

**Reply by giaari15** [reply]
I like how you emphasized the financial gain as a source of unethical practices. Academia and research as we know it in the modern day is so intertwined and rooted in capitalism to where the purpose of research has strayed far from human aspects and community gain to violent profiteering. Due to this, I feel it connects back to the idea of objective vs. subjective conceptions of mapping because if financial gain is the goal, the map itself is then created with biases and is skewed for profit.

**Reply by Keiya-Wada** [reply]
I completely agree with what you and the person who replied before me said. In today's world, almost everything has a hidden(or not hidden) objective to make money off of it. Rather than trying to be completely objective for the sake of honesty and improvement, money makes it easy for authoritative figures to manipulate the data for some sort of benefit. And while the purpose of participatory mapping is to engage the community and improve it, our hidden greed can cause us to completely go against this objective.

**Reply by kianaptran** [reply]
Hi Justine! I completely agree, especially in a capitalist economy where financial benefits are valued more than human lives (sadly). Both replies point out how greed and power can turn good-willed endeavors like academia and research into exploitative practices. The only issue is how we would resolve this. Although there is no direct answer, one potential approach is to remove the researcher's influence on the data collection entirely and have only the communities (people providing the data) give what they choose to share, putting at least some power back to the people.

**Comment by pwinnie2026** [comment]
In the reading, Chambers highlights how "we ('educated'  professionals)" are often fixated on a certain way of making "proper" maps and who would be qualified to make those maps. It immediately narrows the idea of map making and cartography, even though maps take shape in numerous ways in a plethora of communities prior to perhaps the more Westernized approach. To be an "educated" professional in itself is not accessible due to deep-rooted inequities in higher education access. Besides that point, Chambers proceeds to highlight the importance of deconstructing this notion in favor of participatory mapping. Even distinguishing between Ground and Paper participatory maps (and GIS as a medium!), is crucial when considering the accessibility of this information and who can participate in the community production of knowledge. How can you use, understand, and update these maps without some level of accessibility? Where mediums like GIS are favored, how can we look into ground mapping to convey the same information? Maps should, I believe, start collaboratively within locals first. If one wants to consider certain cultural sensitives or reduce risk or danger to the community, it is arguably easier and important for community members to participate in this map making. On a related note, this living [countermap ](https://www.libromobile.com/mappingsantaana) against tourist marketing personally comes to mind. Created by Crear Studio's collaboration with two local artists: Roger Eyes R. and Sarah Rafael García, this map of Santa Ana becomes very personal in preserving and sharing the histories integral to these communities. (My favorite part of this map looks at Santa Ana's former Chinatown.) Nevertheless, it is important to check our biases and assumptions for map making; knowledge production and map making by locals should not be delegitimized. What may seem important from an outsider "expert" perspective may not be truly representative or accurate.

**Reply by stinest** [reply]
Your response was incredibly well said!! I think you make an amazing point in reaffirming the importance of considering accessibility in any sort of mapping. I also enjoyed looking through the Mapping Santa Ana site you linked, and think it ties in well with your response.

**Reply by giaari15** [reply]
The accessibility aspect of mapping should fully be emphasized because it is very revealing on who maps are being made for. I would take it a step further of deconstructing even the presence of an "expert" or "educated professional" in making maps with/for locals. beyond being representative or accurate, I feel that presence itself is still a colonial presence, especially in the western scene of academia and research.

**Reply by jasminereyz** [reply]
I also completely agree that your response was very well said and I love how you mentioned that maps should collaboratively start local since this can lessen the power knowledge gap when creating maps! I have never seen a counter map before so thank you for including that, this helps me understand that more people are aware of these biases and are actively working against them :)

**Comment by jasminereyz** [comment]
Participatory mapping & critical cartography are two aspects that will always seek improvement. Participatory mapping is supposed to include community and local knowledge but with anything, it can easily fall to greed. However, it is beautiful when communities work together to create maps to defend themselves against government power. For instance, [MappingForRights]is an organization that helps at-risk communities in the rainforest demonstrate their presence to governments. This can be crucial when governments decide to tear down rainforests for industrial business. (https://www.mappingforrights.org/participatory-mapping/). Though critical cartography examines these relationships between technology and power, I immediately thought of how Mariah mentioned that digital maps purposely hide information from their viewers. No, we do not know how many people have died on the street that is being shown and no we do not know of a country that was deliberately removed from a map years ago. I never realized how much power maps inherently had until I realized how much I depend on maps, every time I get into a car myself, I go straight to Apple maps, even when I know where I am going sometimes.  I believe it would be very difficult to improve how one views a map because it can vary from person to person, but those who are creating the map have the power to imply anything they please. In all honesty, I have no clear answer or suggestion for improvement, but I hope that even we in this class can educate others on the risks of mapping!

**Reply by Keiya-Wada** [reply]
I completely agree with what you said. Maps are a very important aspect of our lives, and we depend on it a lot for navigation purposes. However, even though we use it so much, we never really think of the potential implications that underlie maps. Like you said, the people creating the maps can choose what to show on the maps and what not to show on the maps, making it a potentially easy tool to manipulate the users. However, as you said, when used right, it can make a significant impact on whatever it is that we are trying to address!

**Reply by kchhoeu** [reply]
Hi Jasmine! You bring up a really good point on the power of maps in one's life. For me, since I am abroad maps are really important to how I navigate the country I am in especially on public transportation. I never really thought much about who controls the maps or input the data, but after reading your post and the discussion from class I think it is also important to educate others of the risks of mapping!

**Comment by Keiya-Wada** [comment]
Participatory mapping and critical cartography can be improved by first getting rid of our fixed mindset that we have on maps, and rather, accept it as a more multifaceted field. When the guest speaker showed us a number of "maps," the majority of the class only chose the world map as a map when in reality all of them were maps. In our current society, maps shape the fundamental base of how we navigate through our lives, as we use applications like google maps every single day. This is most likely also the reason why the majority chose the world map as the only map, because that is the form of maps that we are the most familiar with, and many think of that kind of form to be the only maps used in the world. However, as explained by our guest speaker, all of the shown images were examples of maps. We have to get rid of the fundamental mindset of maps and learn to be more flexible, that there are many different forms to represent geography. In the reading, youths were asked to draw a map to show where waste bins should be located, but her map got rejected by her supervisor and was told to do a proper one. In this example, what the youth draw didn't fit the supervisor's definition of a "proper" map, a definition shaped by his years of experience studying maps. However, no matter how many years one study the maps, this does not overcome the knowledge the youths have on their own town/village, where they live and navigate through every single day. In this way, the "educated" professional chose to reject the maps of waste bins drawn by the youths, which could have potentially included very valuable information, but was most likely rejected because it didn't look good, or because it didn't fit his taste of a "proper" map. As cartography is a developing field, we should get rid of the fixed mindset, and instead embrace its multifaceted nature and include multiple perspectives on a map.

**Reply by kianaptran** [reply]
Hi Keiya! This is such a great point. I also think that the closed-minded perception of cartography dramatically limits the possibilities of using participatory mapping and critical cartography. When we only have the highly colonial, eurocentric maps in mind, we prevent ourselves from stepping outside of those boundaries and expanding our knowledge of other cultures and their diverse maps and knowledge.

**Reply by zhamic7** [reply]
I agree that with the widespread use of applications like Google Maps created and maintained by mostly one corporation, it gets harder and harder for other equally valid maps to be recognized as valuable depictions of information. Any form of spatial representation will leave out some facet of reality and set of perspectives, so I agree that the fixed mindset that Eurocentric maps are the only "proper" maps should be discouraged as it ends up greatly limiting how we are able to perceive the world around us.

**Reply by sachigill** [reply]
I completely agree with you about the fixed mindset regarding maps; there are so many other kinds of maps that don't get proper validation due to the limited view of what is a map. Western/colonial type of maps are not the only maps to exist, especially in regards to participatory mapping and critical cartography. Outside researchers should not impose their view on maps onto local communities, as it is the locals themselves that know what is the most ideal and accessible way to represent their information, for their communities.

**Comment by kianaptran** [comment]
Of the many ethical issues highlighted in the Chamber's article on Participatory Mapping and Geographic Information Systems, one specific issue that stood out was "extracting information which will be used against people" (Chambers 7). This issue initially caught my attention because it was the only issue that did not have a direct example tied to it. However, it did relate somewhat to a concept discussed during the lecture with Mariah and how oral maps have been historically practiced, but the choice to share the translation/meaning prevents the information from being exploited by others (the privilege of understanding the oral map shows trust in the recipient that they would not misuse the information). This is all to say that an issue with participatory mapping and critical cartography is controlling who gets a hold of the information/data so that the information is not used against people and puts people in danger (another ethical problem that Chambers points out as a separate point). To solve this issue, the participants who provide the information must have control and knowledge of where their information is going and who it will be used by to mitigate potential exploitation (much like how oral maps in a native language can only be used by those who understand that language, allowing the information to stay in that culture or be shared/translated to only select few).

**Reply by kchhoeu** [reply]
Hi Kiana! I really think you brought up really good idea in your post! I agree with your solution to solve the issue. I think language is such a important tool that is easily overlooked when it comes to research and data collection, but having participants be able to control the knowledge of their information is important because they deserve a full choice, and also for language to be accessible for the participants.

**Reply by sjeyakum** [reply]
I like how you connected this issue to the point our guest speaker brought up in last week's lecture about control over information and the potential for exploitation when working with any community. I think it's also crucial to recognize that while direct partners and collaborators may strive to protect sensitive information about communities, the institutions and networks they are part of may leverage collected information for inappropriate reasons (i.e., capitalist benefits, government policies, etc.). I think it's fair to say that there can be a significant imbalance of power at all levels in participatory mapping and critical cartography, which makes it more challenging to ensure that those who provide valuable information in the first step retain the most control and knowledge over the collected information. However, I agree that this is an important goal to work towards, as it would also help alleviate other related issues Chambers brings up, such as putting people in danger, as you mentioned.

**Comment by kchhoeu** [comment]
The Chamber article mentions many aspects of mapping and cartography. For participatory mapping, a lot of the experiences could lead to misuse or exploitation. Since it is primarily made to be of use to the community and provide community insight, it has a lot of ethical issues as stated in the Chamber article, "exposing people to danger" (Chamber,7). Though created to help communities it can also show the vulnerabilities of said communities leading to more problems than improvement. To solve the issues brought up, it should allow those who are part of the community to correct the information researchers gathered and also allow the participants to pull out on the data collection and give them more control in the choices. It should also tell the participants the truth of where the information is going and if there are any changes keep them informed.

**Reply by zhamic7** [reply]
I also agree that it is important for researcher to give the communities they were gathering information follow ups regarding changes that have been made as how the data is used should not be solely determined by the data collectors but, in large part, by those who supplied the data. Failing to update the participants is essentially lying to them about the purpose for data collection.

**Reply by sjeyakum** [reply]
I feel like your point about participatory mapping revealing vulnerabilities resonates with what Mariah shared in last week's guest lecture about power and control! I appreciate that your solution involves heightened community engagement and control, ensuring that community participants remain active throughout the mapping process and study—I feel like this is often neglected in a lot of study designs. This method also aligns well with the partnership, transformative, and self-mobilizing participatory mapping approaches in the participation ladder that Chambers includes at the end of his paper. I also think that beyond simply correcting information and gaining more control over it, participants can feel more empowered and represented, as they get to tell their story and history in a way that best encapsulates their identity, rather than having an image constructed by outsiders who simply extract and leave with their information.

**Reply by sachigill** [reply]
I definitely agree with your points! There needs to be more transparency in these processes since ultimately the communities are the ones being affected, not the outsiders.  Allowing communities to correct information helps to bridge the uneven power dynamic between them and outside researchers, and have their voices be heard throughout all parts of the mapping journey.

**Reply by kimada62** [reply]
I agree. While transparency with any form of data collection should be a given, there's not always a lot of focus on the follow ups. While it is different, following up and correcting stories is a part of a journalists duty in sharing the truth. The same should be true for mapping. If something geographically changes, you should update it. If the communities who reside there notice these changes, they should have the ability to make sure the maps remain accurate.

**Comment by zhamic7** [comment]
Participatory mapping and critical cartography would improve if there were increased emphasis on collaboration and forming long-term genuine ties with the group of interest. Although it has been recognized that having locals create their own maps results in maps being made much more efficiently and with way more detail, as observed in Chamber's paper, I feel as if there is still an overwhelming sentiment that participatory mapping involves some outside group suddenly showing up at the local community, requesting that the information needed be shared by local residents, and then leaving soon after. For example, many mappers have ended up taking away time from local communities during critical periods of the year or placing individuals in danger by not realizing the sensitivity of the information shared, which are all a result of not taking the time to get to know the local community and its social dynamics before trying to collect geographical data. Most notably, Chambers notes that there are also cases of repeating activities where more accessible areas are "'carpet-bombed' with PRA... while more 'remote villages' are never visited," indicating a lack of maintenance or follow-ups regarding the information taken. While it is usually better for communities to document information themselves, it is not necessarily always a negative thing for communities to work together instead. However, for the latter to be even mildly successful, it is essential that not only the questions listed in "Ownership and Use" of Chamber's paper to be continuously considered, but also for groups to view themselves as equal collaborators connecting in the long-term for a mutually beneficial relationship rather than some powerful and knowledgeable outside group vs. a weak and ignorant local group dynamic that is much more prevalent currently.

**Reply by nesslee123** [reply]
I completely agree with your response and you brought up many great points! I especially agree with you regarding how there is an emphasis on a more powerful and knowledgeable group coming and going when they see fit because throughout the reading it is reiterated that these researchers would come to these communities, collect the information, and leave, often endangering the people they left behind. I also agree that there is a necessity for a mutually beneficial relationship because it would ensure that marginalized communities are also being considered and cared for.

**Comment by sjeyakum** [comment]
As I was reading the list of ethical issues and abuses of participatory mapping and critical cartography in Chambers’ paper, the point about repeated activities caught my attention. As Chambers explains in his paper, more accessible communities are often sought after to participate in mapping activities by outsiders. This creates a cycle where these communities are repeatedly burdened by participatory mapping activities, while other, more remote communities are neglected and remain unrepresented. This practice not only perpetuates the burden on the frequently visited communities but also exacerbates other abuses Chambers lists, such as raising expectations, causing disillusionment, and extracting information for the sole benefit of outsiders. It is definitely upsetting to see only a few communities get visitors only to have them leave immediately after getting what they want, which does not favor building the trust essential for participatory mapping. Moreover, if this cycle continues without intervention, communities are further oppressed and silenced, despite the paradoxical view that they are receiving many "opportunities" to participate in mapping activities. I feel like this highlights the difference between supporting and truly listening to a community’s needs, as repeated activities show a lack of genuine interest and engagement. While I do not think communities frequently sought after should be taken off any list, I do think that partners or supporters who genuinely want to work with these groups should critically reflect on their approach and understand any shared histories the area has with mapping.There must also be a greater emphasis on voice and representation to ensure empowerment and inclusivity by recognizing that participatory mapping involves listening to and addressing the unique needs of each community, ensuring that all voices are heard and respected.

**Reply by nesslee123** [reply]
Really great and thoughtful response! I completely agree with you in regards to the lack of representation of more remote communities because this repeated burdening of more accessible communities continues to further marginalize more remote communities and emphasizes how these communities are not given an opportunity to participate in map making. Also, this not only places more burden on the more accessible communities, but also continually endangers them as Chambers mentioned in the reading.

**Reply by caitlynn-hauw** [reply]
Perhaps it was because the header "Repeating activities" didn't initially make sense to me, but this point also really stood out to me when I did the reading. I made a comment in my own response about toeing the line between two extremes: maps in isolation and tokenization, which I believe the point you mentioned is guilty of the latter. By "repeating activities," there is an attempt to bridge an inequity by performatively engaging in participatory mapping, but this only creates a further inequity that did not exist prior. As you mentioned, when one voice is continually elevated, empowered, and tokenized—and another voice is silenced—oppression is perpetuated.

**Comment by nesslee123** [comment]
When considering how participatory mapping and critical cartography can be improved, it is important to also consider how this affects marginalized communities. Throughout the reading, I found that it was essential to consider the necessity of the people in these communities and ensure that they are incorporated in the process of map making. Participatory mapping and critical cartography can be improved by this incorporation while also considering the importance of ensuring that academic researchers also participating in map making focus on preventing harmful outcomes to these communities. Chambers (2006) describes, “This has occurred especially with women who take part in participatory activities, and then when the outsider has left are abused or beaten by their husbands” (p. 7). The author also explains, “Street children who made maps of their parts of Cairo in all innocence showed where the drug dealers operated, which could have got them in trouble if the authorities learnt and took action” (Chambers 2006, p. 7). By prioritizing the safety and opinions of marginalized communities in the mapping process, this may lead to increased participation by these communities.

**Reply by pwinnie2026** [reply]
I totally agree with you! The prioritization of both the safety and the opinions/experiences of impacted marginalized communities would give "outsider" researchers the necessary context to not put those children or women (in the examples you mentioned earlier) at risk once the "outsiders" leave. It is in the name for communities to be a part of this participatory map making, and in doing so, these maps are more accurate to the lived experiences of their communities and in respect for their safety. I like to tie it back to the last lecture as well as talking about how to not extract information out of communities but rather create with them.

**Comment by sachigill** [comment]
Participatory mapping and critical cartography could be improved by increasing emphasis on community engagement, and creating environments where these communities feel like their voices are being properly represented during the map making process. There needs to be a fairer power dynamic, as well as more equal collaboration, between local groups and mappers. When it comes to data collection or map creation, there also needs to be more transparency in these processes, such as how their data will be used, who will get access to these maps, etc., to make sure that at the end of the day, the communities themselves are being prioritized, not the outsiders or people in power. We also need to understand that maps don't just fall under one box. Our guest lecturer Mariah talked about how maps are not just the typical world map we see, and so many cultures' maps may differ from what the Western view of a map is, but that doesn’t mean these other forms are not maps. I think understanding the community and what kind of maps they use is crucial. Instead of trying to use certain technologies or create specific maps that we are used to, it is important to take into account their culture and the way local communities represent things and create maps, so that way they can actually be used and understood by the community.

**Reply by NellieNguyen149** [reply]
I completely agree with you! I also discussed a similar idea of emphasizing community engagement to allow their voices to be heard. I also agree with your comments about the current imbalance in power dynamics, and I specifically focused on formal educational levels and how that may play a factor in how maps are created. One factor that could play into the power imbalance is that formally educated individuals do not always have faith in others to produce supposedly "accurate and quality" maps. However, once mapping/educated experts recognize this misconception and give space to everyone, we could then create maps that are more accurate and representative of all communities! Thank you for sharing your response :D

**Comment by NellieNguyen149** [comment]
One idea I have to improve participatory mapping and critical cartography is to create more opportunities for communities to have a say in how their area/living space is documented and mapped. I noticed an example idea that the author of the article, Robert Chambers, mentions on page 3, "...I spent much time being, as I thought, rather clever, filling in a map of much of the Northeast of Kenya by asking pastoralists... it never occurred to me to ask them to draw the map themselves!" Chambers acknowledges that he also has limited perspectives and insights he overlooked in past experiences, which he spends a good portion of his article discussing. However, this idea that he formulates in hindsight of his experience is representative of what I envision for improvements in PMs.

Map creators who come from more formally educated and academic backgrounds tend to oversee the power of first-hand experience when it comes to documenting and mapping. There is another example provided, in this case, maps/documentation made by children were overlooked, "In Kingston, Jamaica, in the 1970s, Frances Madden (pers comm.) asked youths to draw a map to show where waste bins should be located; but when she showed the map to her supervisor he told her to go away and do a proper one... But in general, we (“educated” professionals) were so fixed on our own cartography and ideas of what were “proper” maps, and on the belief that only “we” could make them, that we did not realise how well and how useful local “uneducated” people did, and could, make their own." Beyond the youth, some experts seem to have a misinformed belief that less formally educated people are not capable of producing accurate or "up to standard" maps, when this isn't the case! 

Living in the mapped area can provide major insights, along with decreasing potential inaccuracies and human errors compared to when information is communicated through another party. Even without access to formal education, people can be fully capable of forming maps of what they experience for themselves, thus providing a true form of individual perspective and thought. This allows for a more compelling shift of the term opening up "invited spaces" that Robert Chambers mentions on page 2. 

Instead of those in a position of authority when forming maps to simply claim spaces without invitation (automatically assuming they have a right to), there is an opportunity for those who were not fortunate enough to be in a more "powerful" position to voice/map their experiences and knowledge. While the whole idea of being in a "position of power and authority" in itself displays a power imbalance and injustice, putting more emphasis on creating "invited spaces" can possibly mitigate some of the challenges our current system creates.

**Reply by caitlynn-hauw** [reply]
I really enjoyed Chambers' mention of the nuances in even the words we use to describe mapping. I think he did make a distinction, however, between spaces that community members are invited to and spaces that community members claim. When a space has the ability to be claimed by the community vs. when a community is invited to a space, I think it emphasizes the idea of ownership and who owns what. With an invitation, there is an inherent power between the inviter and the invitee; whereas, the community that claims has the power to claim for themselves.

**Comment by caitlynn-hauw** [comment]
With the advent of ever-advancing GIS technologies, many have a tendency to focus on the capabilities of the technology rather than the capabilities of the community members these technologies should be serving. In choosing to put words like "educated" and "uneducated" in quotes, Chambers emphasizes the subjectivity of the words. Is the more "educated" more educated than the "uneducated," or do they merely wield the power to create that distinction? Our past maps have reflected an ignorance and arrogance of our perceived superiority over community members who are more skilled and best equipped to serve themselves than we ever will be. 

Currently, modern-day mapping can be inequitable. It can require a barrier-to-entry that makes participatory mapping nearly impossible. When doing participatory mapping and critical cartography, it's important to ask many of the questions Chambers raised, such as who participates in whose mapping, where is it kept, who owns the map, who is empowered and who is disempowered? We must toe the line between creating work in silo, as an outsider, and tokenizing local community members. Our responsibility as mappers is not to make a map to save a community but to allow a community to save themselves, with the belief that they are more than capable of doing so. It's a recognition that the technology we wield is a privilege and a power—that comes from our own privilege and power—but will only be useful if we empower the communities to map themselves and own their own maps.

**Reply by NellieNguyen149** [reply]
I think you bring up an amazing point! After reading your response, I realized how reliant we can be on the technology aspect of mapping that it can inadvertently affect the full accuracy of what we're trying to represent. I also agree with the acknowledgment you made about how modern-day mapping has a barrier-to-entry (access to technology/related resources) that makes it near impossible to produce maps that others would reference and appreciate. Thank you for sharing! :D

**Reply by kimada62** [reply]
I agree with your barrier to entry comment. I think we need to lower that barrier for entry by having those with access to the maps reach out to the people who live there and have a cultural history there. We need to lose our egos and savior complexes in order to open the door for others.

**Reply by chrissanrow** [reply]
I like how you criticized our perhaps overestimation and appraisal of the powers of cartographical tools. They are in fact, simply tools, and their real value comes from the people who use and produce with them. Consequently, solely relying on technology to solve problems is insufficient, which is a great fault that many people looking to evoke change with these tools have unfortunately engaged in. Thus, I think your ideas of not "saving" these communities with technology but enabling them to utilize technology for themselves in whichever way they see fit is critical.

**Comment by kimada62** [comment]
Our people were for a long time ignorant of local people's mapping abilities. Participatory mapping and critical cartography was therefore lacking because outsiders would attempt to map out an area they knew little about. They thought that just walking and talking with locals would be sufficient information on the geography. In the reading they made the comparison of their inaccurate well mapping to the locals' 25 minute recall of all the wells locations and conditions. These "Eureka! moments" (page 3) should be very clear in showing how much us foreigners can rely on the people who live in these areas for participatory mapping. By assuming that they will not be able to contribute just because of our western educations is ignorant and actually stops us from learning. We need to draw things out with a stick and invite locals to participate without biases toward the "educated" and "professionals," who defined as such by cultures that do not reside in the areas of study.

**Reply by chrissanrow** [reply]
I completely agree with your sentiments here and think that we could go even further by not just inviting them to participate but to engage and empower them to produce solutions for themselves!

**Comment by chrissanrow** [comment]
Participatory mapping, critical cartography, and GIS, as discussed by the reading have been historically gatekept to the hands of the educated and the elite. Attempts to utilize cartography to empower marginalized communities have only recently begun shifting towards a more critical and hands-off approach. The error in previous endeavors lies in the assumptions of inability and a near savior-complex, causing educated individuals to ignore possible more inclusive approaches, such as requesting local individuals to create the maps when working to map out a particular area, as mentioned in the reading. Not only is participatory mapping contingent on requesting the consent of local communities, but it should also prioritize taking on a supportive role for these marginalized groups. The benefits of this are numerous: individuals of these communities know their needs and circumstances best, allowing them to potentially produce more meaningful and practical solutions otherwise unlikely to have been thought of by the hands of foreign individuals or groups.

Ultimately, a multitude of considerations must be made when approaching participatory mapping and critical cartography, including whether such efforts are actually beneficial for individuals of the community. When a project is led from an outsider group, it is often too easy to adhere to a savior-like mindset, disregarding the capabilities of the community and even, in the worst cases, utilizing the group's consent for personal gain.
